backend/environment.py

```diff
@@ -1,3 +1,12 @@
 import os
 
+from aws_cdk.core import RemovalPolicy
+
 ENV = os.environ.get("DEPLOY_ENV", "test")
+RESOURCE_REMOVAL_POLICY = os.environ.get("RESOURCE_REMOVAL_POLICY", "DESTROY")
```

Comment:
We can inline this, because the value is only used in one place.

Reply:
Inlined now.